io/dune

```
@@ -12,7 +12,7 @@
    (:include c_flags.sexp)))
  (c_library_flags
   (:include c_library_flags.sexp))
- (libraries odiff-core camlimages.jpeg camlimages.tiff camlimages.xpm))
+ (libraries odiff-core camlimages.png camlimages.tiff camlimages.xpm))
```

Are you sure, this fixes the current linking issues?
I think with this change we are introducing the dynamically linked `libpng` again and are still depending on `libXpm` and `libtiff`.

I would suggest removing camlimages completely (as the title says) 
```suggestion
 (libraries odiff-core))
```

...or did i misunderstand the goal of this pr?

---

It's neat but we will need to remove the camlimages IO then and now it will at least try to use it and fail with unsupported image format exception. But yeah I agree we can do that

---

To temporarily remove support for `xpm`, `jpeg` and `tiff`, we should probably remove the complete CamlimagesIO from odiff.

---

Ok I will push it. But what do you think – should we release 3.0 with breaking change or not?

---

I kind of see it like you do. 
It currently doesn't work, so it isn't really a breaking change. I personally would not release it as a new major version :)